android/engine/src/main/java/org/smartregister/fhircore/engine/ui/appsetting/AppSettingViewModel.kt

```diff
@@ -144,9 +146,18 @@ constructor(
       showProgressBar.postValue(false)
     }
       .onFailure {
-        Timber.w(it)
+        Timber.i("Failed to load configs: Error [${it.localizedMessage}]")
```

Comment:
Use Timber.e instead. This is an error log.

Reply:
Updated.